cmake/FindCUDA.cmake

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+message(STATUS "Checking for module 'cuda' using find_library()")
+
+find_library(CUDA_LIBRARY NAMES libcuda cuda)
+set(CUDA_LIBRARIES ${CUDA_LIBRARY})
+
+get_filename_component(CUDA_LIB_DIR ${CUDA_LIBRARIES} DIRECTORY)
+set(CUDA_LIBRARY_DIRS ${CUDA_LIB_DIR})
+
+if(WINDOWS)
+    # find_file(CUDA_DLL NAMES "bin/cuda.dll" "cuda.dll")
```

Comment:
???

Reply:
right, uncommented. BTW - the Windows is not tested yet (we don't have proper machine in CI)